dist/src/main/resources/log4j2.component.properties

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+# one or more contributor license agreements. See the NOTICE file distributed
+# with this work for additional information regarding copyright ownership.
+# Licensed under the Zeebe Community License 1.1. You may not use this file
+# except in compliance with the Zeebe Community License 1.1.
+#
+# Set up garbage free logging; see https://logging.apache.org/log4j/2.x/manual/garbagefree.html
+log4j2.enableThreadlocals=true
+log4j2.enableDirectEncoders=true
+log4j2.garbagefreeThreadContextMap=true
```

Comment:
Nice TIL 👍🏼 why is this not a default 😅

Reply:
Yeah, I found out in a blog post of all places and was wondering why we didn't do that :smile: